### ISSUE TITLE
[QUESTION] From (SRTM -> .DAT) to (GeoTiff - >.DAT) conversion

### ISSUE THREAD

**Issue by yaroslavkulinich** [ArduPilot/terraingen#6]
Hello.

As I see there's the only place where I can understand the elevation data format for ArduPilot offline usage. 
For using custom elevation data source (from GeoTiff files) our team needs some sort of converter to ArduPilot .DAT format.

The basic question for understanding.
Is it possible to make GeoTiff to ArduPilot DAT elevation format converter based on Python logic from this repo? Or maybe there are some gaps for this (excluding GeoTiff side)? 

Thank you.

**Comment by stephendade** [comment]
>  Is it possible to make GeoTiff to ArduPilot DAT elevation format converter based on Python logic from this repo? Or maybe there are some gaps for this (excluding GeoTiff side)?

Yes, you should be able to. Note that this repo inherits some logic from the MAVProxy repo (https://github.com/ArduPilot/MAVProxy)

**Comment by Ryanf55** [comment]
I am considering contributing a GDAL driver for the ArduPilot tile format. It would make creation of Ardupilot tiles significantly easier.
